### PR TITLE
[Bug] mobile responsive menu search

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
the search bar is adapted for mobile and desktop version
Bildschirmfoto vom 2019-07-04 08-22-19

 

# Issues
fixes  #https://github.com/Human-Connection/Human-Connection/issues/975

### CODE REVIEW

**Comment by mattwr18** [comment]
I have added a new mobile responsive navbar in my `PR` https://github.com/Human-Connection/Human-Connection/pull/969
I didn't want to step on any toes, but as we add more to the navbar, the need to make it more mobile-friendly increases... please let me know what you think @roschaefer @ogerly

File: webapp/layouts/default.vue [hunk 1]
```diff
@@ -192,7 +192,9 @@ export default {
       return this.$route.path.indexOf(url) === 0
     },
     unfolded: function() {
-      document.getElementById('nav-search-box').classList.add('unfolded')
+      if (window.innerWidth < 765) {
```
[the remark this answers is not in the document]
Comment (ogerly) [reply webapp/layouts/default.vue]: ok

**Comment by ogerly** [comment]
@mattwr18  very good matt, I also think that this will become even more extensive. 
I would have also gladly built a new one. ;) 
i close this PR and refer to your PR @mattwr18

**Comment by ogerly** [comment]
dieses PR ist veraltet und wird geschlossen. hier ist das aktuelle zu dem thema: https://github.com/Human-Connection/Human-Connection/pull/969

**Pull request closed without merging**